envconfig/config.go

```diff
@@ -235,6 +235,25 @@ var (
 	MaxVRAM = Uint("OLLAMA_MAX_VRAM", 0)
 )
 
+func Uint64(key string, defaultValue uint64) func() uint64 {
+	return func() uint64 {
```

Comment:
This can be in a future PR, but why do we return a closure here. Wouldn't this always be a fixed value since it's coming from an env var?